src/sagemaker_pytorch_container/training.py

```diff
@@ -96,6 +96,13 @@ def train(training_environment):
         capture_error = False
         logger.info(f'capture_error is {capture_error}. Default is True')
 
+    try:
```

Comment:
can we write a function like _set_nccl_environment function above?
like _set_torch_version_environment

Reply:
Yes.